android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationProxy.java

```diff
@@ -183,7 +187,16 @@ public void setLargeIcon(Object icon)
 		}
 		setProperty(TiC.PROPERTY_LARGE_ICON, icon);
 	}
-
+    
```

Comment:
Remove spaces from empty lines - here and on line 199.

Reply:
Done